examples/ssm-examples.h

```diff
@@ -49,15 +49,15 @@ void ssm_throw(enum ssm_error reason, const char *file, int line,
 }
 
 #define MAX_PAGES 2048
-static void *pages[MAX_PAGES];
+static uint8_t *pages[SSM_MEM_PAGE_SIZE][MAX_PAGES];
```

Comment:
Mem pool pages are now allocated out of a static megapool now, so as to not confuse Valgrind.